source/base/Text.ooc

```diff
@@ -174,6 +174,14 @@ Text: cover {
 		this free(Owner Receiver)
 		result
 	}
+	print: func {
+		printf("%s", this _buffer _backend pointer as Char*)
+		this free(Owner Receiver)
+	}
+	println: func {
+		printf("%s\n", this _buffer _backend pointer as Char*)
```

Comment:
Yeah you never know without looking at C code. There was a bug related to this in plotting module not so long ago.